fapolicy_analyzer/glade/searchable_list.glade

```diff
@@ -98,14 +98,15 @@
             <child>
               <object class="GtkFrame">
                 <property name="visible">True</property>
-                <property name="can-focus">False</property>
-                <property name="label-xalign">0</property>
-                <property name="label-yalign">0</property>
-                <property name="shadow-type">in</property>
+                <property name="can_focus">False</property>
+                <property name="vexpand">False</property>
+                <property name="label_xalign">0</property>
+                <property name="label_yalign">0</property>
+                <property name="shadow_type">in</property>
```

Comment:
I think the file changes here are related to PR #491 not this PR. If that's the case please revert this file.